### PR TITLE
migrate from 'flutter pub run' to 'dart run'

### PR DESCRIPTION
**Pull request by freemansoft**
"flutter pub run" has been obsoleted. Updating docs.

### CODE REVIEW

**Comment by freemansoft** [comment]
Is there something I need to do in order to get the "Add new issues to project" task to run?

File: examples/marvel/README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Where `public_key` and `private_key` are obtained from https://developer.marvel.
 Since this project uses [freezed](https://pub.dev/packages/freezed) for code generation, be sure to run the generator with the following command before attempting to build/run the application:
 
 ```sh
-flutter pub run build_runner build
+dart run build_runner build --delete-conflicting-outputs
```
Comment (rrousselGit) [inline examples/marvel/README.md:48]:
```suggestion
dart run build_runner build -d
```

File: examples/pub/README.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 ```
 
-Build the riverpod packages
+## Build the Example App
 
+Run the code generation
 ```bash
+cd examples/pub
 flutter pub get
+dart run build_runner build --delete-conflicting-outputs
```
Comment (rrousselGit) [inline examples/pub/README.md:16]:
```suggestion
dart run build_runner build -d
```

File: examples/stackoverflow/README.md [hunk 3]
```diff
@@ -11,5 +11,5 @@ This can be done with:
 
 ```sh
 cd examples/stackoverflow
-flutter pub run build_runner build --delete-conflicting-outputs
+dart run build_runner build --delete-conflicting-outputs
```
Comment (rrousselGit) [inline examples/stackoverflow/README.md:14]:
```suggestion
dart run build_runner build -d
```

**Comment by rrousselGit** [comment]
LGTM, thanks

**Comment by freemansoft** [comment]
I like examples that are a little more verbose and would have left the long command line parameters.  They make it easier to learn.

**Pull request merged**